### ISSUE TITLE
Call Center receives message about Intake change when the "Cancel" button is clicked

### ISSUE THREAD

**Issue by drodruck** [josephr5000/confdb-public#257]
ENVIRONMENT
Windows 10 Pro, Firefox
ISSUE
Call Center receives message about Intake change when the "Cancel" button is clicked
WHAT PAGE
Intake: Status & Messages 
WHAT STEPS WHERE TAKEN
Entered bogus data in Expected wait field, got error message, changed the entry to proper data, then clicked the “Cancel” button
WHAT I EXPECTED
No message to the CC
WHAT HAPPENED
CC received a message that the conference had changed the intake settings.

**Comment by josephr5000** [comment]
Curious: did the settings actually get changed, in which case the notification was right but the cancel button didn't cancel?  Or nothing changed, in which case cancel worked but notification was unnecessary?

**Comment by drodruck** [comment]
The settings were changed but the cancel button didn't cancel.

**Comment by josephr5000** [comment]
Very helpful, thanks!

**Comment by josephr5000** [comment]
NOT REPRO.  Tried on all 3 intake screens.  Entering bogus data gave a validation error (expected), then correcting that and pressing cancel actually cancelled and didn't modify a thing (also expected).  No notifications were sent at any time.

Just to be sure, I went into one of the intake settings and actually made a change and pressed *save*.  The change *was* saved, and I did get a notification.

Ideas?  Thanks.
